monai/eval_savers/nifti_saver.py

```diff
@@ -0,0 +1,103 @@
+# Copyright 2020 MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import numpy as np
+import torch
+from monai.data.nifti_writer import write_nifti
+
+
+class NiftiSaver:
+    """
+    Save the data as nifti file, it can support single data content or a batch of data.
```

Comment:
need some docstrings about how to use this class's methods

Reply:
Updated in the latest patch.
Thanks.